.cirrus.yml

```diff
@@ -335,7 +335,7 @@ static_alt_build_task:
         - build
     # Community-maintained task, may fail on occasion.  If so, uncomment
     # the next line and file an issue with details about the failure.
-    allow_failures: $CI == $CI
+    # allow_failures: $CI == $CI
```

Comment:
Do we want to disallow failures now?

Reply:
I *think* so.  I did this on master as well.  The switch to cachix seems to have greatly improved the reliability of this job.  I'm puzzled why Cirrus's cache failed us...but "proof of the pudding is in the eating" :shrug: